diff --git a/node_modules/react-native-camera-kit/src/CameraScreen.tsx b/node_modules/react-native-camera-kit/src/CameraScreen.tsx
index 4737bc3..4a93abb 100644
--- a/node_modules/react-native-camera-kit/src/CameraScreen.tsx
+++ b/node_modules/react-native-camera-kit/src/CameraScreen.tsx
@@ -396,10 +396,9 @@ const styles = StyleSheet.create(
       alignItems: 'center',
     },
     bottomButton: {
-      flex: 1,
+      flex: 0,
       flexDirection: 'row',
       alignItems: 'center',
-      padding: 10,
     },
     bottomContainerGap: {
       flex: 1,
